doc/classes/Tween.xml

```diff
@@ -13,7 +13,7 @@
 		        Tween.TRANS_LINEAR, Tween.EASE_IN_OUT)
 		tween.start()
 		[/codeblock]
-		Many methods require a property name, such as "position" above. You can find the correct property name by hovering over the property in the Inspector. Many methods require a property name, such as "position" above. You can find the correct property name by hovering over the property in the Inspector. You can also provide the components of a property directly by using "property:component" (eg. "position:x"), where it would only apply to that particular component.
+		Many methods require a property name, such as "position" above. You can find the correct property name by hovering over the property in the Inspector.  You can also provide the components of a property directly by using "property:component" (eg. "position:x"), where it would only apply to that particular component.
```

Comment:
There are two spaces after "the Inspector.", it should only be one.
Could you also change the quoted strings to use `[code]position:x[/code]` instead, so that they appear as inline code?
And squash the commits together.